Mention about NuGet.PackageSourcemapper tool for package source mapping feature (#2844)
NuGet/docs.microsoft.com-nuget: docs/consume-packages/Package-Source-Mapping.md

### Get started

> To fully onboard your repository you can do it 2 ways either manually or automated tool.

Reviewer: I think the sentence would flow better if the order is flipped,

```suggestion
There are 2 ways you can fully onboard your repository, manually or using the NuGet.PackageSourceMapper tool.
```

Reviewer: We can also make them linkable but not a big deal.

### Get started

> To fully onboard your repository you can do it 2 ways either manually or automated [NuGet.PackageSourceMapper tool](https://www.nuget.org/packages/NuGet.PackageSourceMapper/0.1.3-preview.22218.1#readme-body-tab), for manual onboarding you may take the following steps.

Reviewer: I'd just create 2 different groups, potentially under subheadings such as `manual onboarding` and `automated` onboarding. 

It's also a good idea to add more details about what the tool deos and how it'd be different from manual onboarding. For example, the tool only adds package ids.

https://devblogs.microsoft.com/nuget/quickly-map-your-nuget-packages-to-sources/#get-started has some good comments, we should try to write a few sentences.

It's also probably a good idea to link https://www.nuget.org/packages/NuGet.PackageSourceMapper#readme-body-tab and not link to a specific version instead.

Author: I addressed your comment, please check again.

> To fully onboard your repository you can do it 2 ways either manually or automated [NuGet.PackageSourceMapper tool](https://www.nuget.org/packages/NuGet.PackageSourceMapper/0.1.3-preview.22218.1#readme-body-tab), for manual onboarding you may take the following steps.

Reviewer 2: How about a note here?

https://docs.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning

Also let's just link to the package, not the version nor readme. that way we dont have to update the docs with versions.

Author: I improved the text based on feedback from you and Nikolche, please check again.

    * Simplifying the configuration to fewer declarations by using broader package ID prefixes or [setting a default source](#setting-default-sources) where possible.
    * Verifying the source each package was restored from by checking the [metadata files in the global packages folder or reviewing the restore logs](https://devblogs.microsoft.com/nuget/performance-and-polish-with-nuget-5-9/).

#### Automated onboarding using tool

> For large solution with a many packages above manual onboarding could be time consuming and error prone. We developed [NuGet.PackageSourceMapper tool](https://www.nuget.org/packages/NuGet.PackageSourceMapper) automatically generate a NuGet.config for you based on your project’s known packages and sources. The package source mapper tool requires you to have completed a successful package restore in which it will read each respective `.nupkg.metadata` file generated as part of your build to best understand how you map your respective packages and sources. Tool not only covers top dependencies it also considers all the transitive dependencies when generating mapping.

Reviewer: Consider entering a new line after the second sentence (basically before you have the text from the blog).  That'd improve readability as it doesn't look like a complete block.

The first 2 sentences have some small errors. 

How about

> Many repositories have a large number of packages and doing the work manually can be time consuming. The [NuGet.PackageSourceMapper tool](https://www.nuget.org/packages/NuGet.PackageSourceMapper) can automatically generate a NuGet.config for you, based on your project's known packages and sources.

Author: Sure, addressed 2 PR comments.
